CrashReporting/Sources/Domain/Services/CrashReportingServiceAdapter.swift

```diff
@@ -8,28 +8,20 @@
 import Foundation
 import Logger
 
+/// This implementation of CrashReportingService starts the APIService
+/// when a botLogHandler is set. So be sure to always set it.
```

Comment:
I don't think I understand this comment. It references `botLogHandler` but `botLogHandler` was removed?

Reply:
I forgot to remove that comment, it was useful in the previous iteration when that was true because I wanted to start bugsnag after that handler was set otherwise I couldn't hook on the events from the previous session that were sent at the start, but I could refactor it so we don't need that anymore.